src/styles/Book.css

```diff
@@ -0,0 +1,105 @@
+.main-div {
+  display: grid;
+  grid-template-columns: 2fr 2fr 2fr;
+  padding-left: 1.288rem;
+  margin: 0.375rem 6.25rem 0.125rem 6.25rem;
+  background-color: #fff;
+}
+
+.title {
+  height: 1.813rem;
+  margin: 0.188rem 0 0 0;
+  font-family: Roboto, Helvetica, sans-serif;
```

Comment:
- [ ] It would be better if we implement the CSS DRY principle , our file has a few repeating rules which needs to be fixed. For example; ‘ font-family: Roboto, Helvetica, sans-serif’, ‘display: flex’, ‘font-weight: bold’, ‘  align-items: center’, . Kindly check for both CSS files